force-app/main/default/classes/DMLOperations.cls

```diff
@@ -145,9 +169,36 @@ public with sharing class DMLOperations {
      * @param oppNames The list of Opportunity names.
      */
     public static void upsertOpportunities(String accountName, List<String> oppNames) {
+        //Find Account
+        Id accountId;
+        List<Account> accList = [SELECT Id FROM Account WHERE Name = :accountName LIMIT 1];
+                if(accList.isEmpty()){
+                    Account newAcc = new Account();
+                    newAcc.Name = accountName;
+                    insert newAcc;
+                    accountId = newAcc.Id;
+                }
```

Comment:
Mind your indentation.

Reply:
Does that look better? What are the general rules for indentations?

Reply:
Looks better.
I follow a single tab whenever I go inside of code blocks.